scio-tensorflow/src/main/scala/com/spotify/scio/tensorflow/TensorFlowFunctions.scala

```diff
@@ -400,25 +404,37 @@ class TFExampleSCollectionFunctions[T <: Example](val self: SCollection[T]) {
             sys.error(s"kind must be set - feature is ${feature.toString}")
         }
       }
-      .aggregateByKey(MultiAggregator((Aggregator.max[Int], Aggregator.min[Int])))
-      .map { case ((featureName, featureType), (max, min)) =>
+      .aggregateByKey(MultiAggregator((Aggregator.max[Int], Aggregator.min[Int], Aggregator.size)))
+      .withSideInputs(countSI)
+      .map { case (((featureName, featureType), (max, min, size)), ctx) =>
```

Comment:
wondering if we could move the feature building logic to an external function, removing the need for `// scalastyle:off cyclomatic.complexity` (can be done in a separate PR)

Reply:
fixed.